report.qmd

```diff
@@ -51,15 +83,20 @@ Over recent decades, Jablonec nad Nisou has experienced overbuilding, channeliza
 library(leaflet)
 library(sf)
 
-# Read the stream GeoJSON without 
+# Read the stream GeoJSON
 stream <- st_read("Inserted_images/stream_for_interactivemap.geojson", quiet = TRUE)
 
 # Coordinates for Jablonec nad Nisou
 jablonec_coords <- c(lat = 50.7224, lng = 15.1708)
 
-# Render leaflet map
-leaflet() %>%
+# Render leaflet map with zoom control and max bounds
```

Comment:
Nice that you use an interactive map. Optionally, instead of showing a point for Jablonec, you could show its boundaries.